packages/agents/chain-abstraction/src/helpers/asset.ts

```diff
@@ -0,0 +1,66 @@
+import { jsonifyError } from "@connext/nxtp-utils";
+import { axiosGet } from "@connext/nxtp-utils";
```

Comment:
This needs to be imported from mockable functions in order to make the mocks work.

Reply:
https://github.com/connext/monorepo/pull/4324/commits/e4bfb51d8109879f2afe3c8926966f0c8f3a92a4